scripts/triggers-end-to-end-tests/database.rules.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    ".read": false,
+    ".write": false
```

Comment:
do we need rules at all? the emulator starts with open rules

Reply:
It looks like we don't need either `firestore.rules` or `database.rules.json`, but `firestore.rules` needs to be present in `firebase.json` even if the actual file doesn't exist.